docs/installation_windows.rst

```diff
@@ -101,15 +105,6 @@ To install system-wide, run from and Administrator console.
 
 
 
-Install LISA in a virtual environment (Option 2)
```

Comment:
@avylove Are there scenarios where user might have to use this installation?

Reply:
Given the large number of dependencies and the fact most are pinned, unless your environment is only used for LISA, I would expect installing in a virtual environment, either self-constructed or built using nox, is the only reasonable way to install LISA.